core/src/avm2/filters.rs

```diff
@@ -301,16 +310,22 @@ impl FilterAvm2Ext for ConvolutionFilter {
         let preserve_alpha = object
             .get_public_property("preserveAlpha", activation)?
             .coerce_to_boolean();
-        matrix.resize((matrix_x * matrix_y) as usize, 0.0);
+        let mut flags = ConvolutionFilterFlags::empty();
+        if clamp {
```

Comment:
How about converting this to `flags.set(ConvolutionFilterFlags::CLAMP, clamp);` (here and in other places)?

Reply:
Personally I don't feel like that reads as easy as "if this then add this", but if you're sure I can do that change. It's functionally equivalent.

Reply:
I see it cleaner, and it aligns with the rest of the codebase.